src/aragonjs-wrapper.js

```diff
@@ -265,7 +265,7 @@ const initWrapper = async (
     : dao
 
   if (!daoAddress) {
-    console.log('Invalid DAO address')
+    onError('INVALID_DAO_ADDRESS')
```

Comment:
Quick thing: we could make these errors Symbols :D.

Reply:
I’ll open another PR for this!